pkg/infoschema/infoschema_v2.go

```diff
@@ -343,6 +343,18 @@ func (is *infoschemaV2) AllSchemas() (schemas []*model.DBInfo) {
 	return
 }
 
+func (is *infoschemaV2) AllSchemaNames() []model.CIStr {
+	rs := make([]model.CIStr, 0, is.Data.schemaMap.Len())
+	is.Data.schemaMap.Scan(func(item schemaItem) bool {
+		rs = append(rs, item.dbInfo.Name)
+		return true
+	})
```

Comment:
Is it safe for us only fetch schema from cache instead of tikv?

Reply:
Same as AllSchemas()